react-gbajs/src/emulator/gba.js

```diff
@@ -58,14 +58,9 @@ function GameBoyAdvance() {
 	this.seenSave = false;
 	this.lastVblank = 0;
 
-	this.queue = null;
 	this.reportFPS = null;
-	this.throttle = 16; // This is rough, but the 2/3ms difference gives us a good overhead
 
 	var self = this;
-	window.queueFrame = function (f) {
```

Comment:
this is an unrelated change, but switching from `setTimeout` to `requestAnimationFrame` causes gbajs to run at a more correct speed on fast machines. With `setTimeout`, 2/3ms is skipped each frame, causing the game to run just a smidge too fast, and it is noticable. With RAF, gbajs runs at 60fps and feels closer to real hardware.